packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -2827,7 +2888,7 @@ export default abstract class SqlIntegration
                 : ""
             }
             ${
-              data.ratioMetric
+              data.ratioMetric && !data.regressionAdjusted
```

Comment:
The capping and the denominator sum are used for both RA and non-RA ratio metrics. Can't we just rewrite this to just add the new RA fields in a condition within the broader data.ratioMetric block?